package.json

```diff
@@ -37,7 +35,14 @@
     "precommit": "pretty-quick --staged",
     "prepush": "bash ./tasks/changelog-changed-check.sh && yarn lint",
     "postcheckout": "yarn",
-    "watch": "tasks/watch.sh"
+    "watch": "tasks/watch.sh",
+    "nodes": "node ./tasks/testnet.js",
+    "proxy": "caddy",
+    "stargate": "./builds/Gaia/darwin_amd64/gaiacli rest-server --laddr 'tcp://localhost:9070' --home './builds/testnets/local-testnet/cli_home' --node 'http://localhost:26657' --chain-id 'local-testnet' --trust-node true",
+    "frontend": "webpack-dev-server --hot --colors --config webpack.renderer.config.js --port 9080 --content-base app/dist --https --mode=development",
+    "frontend:fixed-https": "yarn frontend --cert 'server_dev.crt' --key 'server_dev.key'",
+    "backend": "yarn proxy & yarn stargate & yarn nodes 3",
```

Comment:
I can't find a way to specify the number of nodes by running yarn start, but can be added back later on in a separate PR to not make this more complex.